designer/server/src/routes/forms/library.js

```diff
@@ -34,10 +37,11 @@ export default [
     path: '/library/{slug}',
     options: {
       async handler(request, h) {
-        const { params } = request
+        const { auth, params } = request
+        const token = auth.credentials.idToken
```

Comment:
Might as well stick with your destructuring?

Presume this one is supposed to be ~`idToken`~ `token` too?

```suggestion
        const { token } = auth.credentials
```

Reply:
Resolved with 6e6ea6ef